src/main/resources/static/app/components/sidebar/lab-requests-sidebar-template.html

```diff
@@ -3,30 +3,46 @@
 This file is distributed under the GNU Affero General Public License
 (see accompanying file LICENSE).
 -->
+<script>
+    function changeFilterStatus(status) {
+        var cur_filter = JSON.parse(localStorage.getItem('labrequests'));
+        console.log('Filter old ' + JSON.stringify(cur_filter));
+        if ('search' in cur_filter){
+            if('predicateObject' in cur_filter['search']){
+                if ('status' in cur_filter['search']['predicateObject']){
```

Comment:
`lodash` has a function `_.get` which takes on object and a path and retrieves a nested value.
E.g., `_.get(cur_filter, 'search.predicateObject.status')`. Then this could be reduced to
```javascript
if (_.get(cur_filter, 'search.predicateObject.status', undefined) != undefined) {
    ...
}
```
See https://lodash.com/docs/4.17.4#get